official/utils/logs/logger.py

```diff
@@ -184,7 +184,8 @@ def _gather_run_info(model_name):
   run_info = {
       "model_name": model_name,
       "machine_config": {},
-      "run_date": datetime.datetime.now().strftime(_DATE_TIME_FORMAT_PATTERN)}
+      "run_date": datetime.datetime.utcnow().strftime(
+          _DATE_TIME_FORMAT_PATTERN)}
```

Comment:
Might be useful to add a get_timestamp() function to avoid having these diverge over time? Two statements of this I can handle, though, so feel free to decide.

Reply:
I think for now I will leave it as is since even it will just be a one line wrap function. I will extract it when it becomes more complicated.